tests/honorCode.test.js

```diff
@@ -41,7 +41,7 @@ describe('Exam assessment response to `requireHonorCode`', function () {
   context.baseUrl = `${context.siteUrl}/pl`;
   context.courseInstanceBaseUrl = `${context.baseUrl}/course_instance/1`;
 
-  before('set up testing server', helperServer.before);
+  before('set up testing server', helperServer.before());
```

Comment:
This whole test suite wasn't being executed. Now that it's included, I had to fix tests that had been broken since the suite was added.

Reply:
Huh!!!